system-addon/content-src/components/Search/_Search.scss

```diff
@@ -2,11 +2,7 @@
   $search-height: 35px;
   $search-input-left-label-width: 35px;
```

Comment:
This should be renamed to something referring to icon. And is the sizing still correct given that the icon size is increasing from 16px to 18px? Perhaps we want something like
```scss
$search-icon-size: 18px;
$search-icon-padding: 8px;
$search-icon-width: 2 * $search-icon-padding + $search-icon-size;
```